enterprise/wsproxy/appstatsreporter.go

```diff
@@ -0,0 +1,21 @@
+package wsproxy
+
+import (
+	"context"
+
+	"github.com/coder/coder/coderd/workspaceapps"
+	"github.com/coder/coder/enterprise/wsproxy/wsproxysdk"
+)
+
+var _ workspaceapps.StatsReporter = (*appStatsReporter)(nil)
+
+type appStatsReporter struct {
```

Comment:
do we expect more structures matching `StatsReporter`?

Reply:
Yes, remove the interface then.

Reply:
Huh? There are two implementations of stats reporter, one in `workspaceapps` (the DB one) and this one (HTTP POST one).

Reply:
I missed the `StatsDBReporter`. I guess that you can mark it that it implements `StatsReport`.

~~Out of curiosity: why are they sharing the same interface? I see only `Report(..., stats []StatsReport)` function, and I'm not sure if it is worth introducing coupling between.~~

Oh I see, this is a workaround to collect data from "wsproxy".